src/components/contact-support/ContactSupport.css

```diff
@@ -0,0 +1,3 @@
+.contact-support-card {
+  max-width: 750px;
+}
```

Comment:
These kind of style tweaks are ones that we want to leave out for now as providing this control within the web-component is something that we should consider, to prevent other teams from having to do the same hacks to match the designs. 

So I think it is best if we remove this style and create a task to allow for max-width in the callout.

Reply:
That was a hardcoded value. In this case we are setting a max-width of the Callout (750px) of the parent container, but it is to restrict the width of the callout. I am saying that maxwidth should probably be a property of the callout to prevent the need for the wrapping container.